app_backend/balance.py

```diff
@@ -0,0 +1,55 @@
+from user import User
+
+
+class BalanceTracker(User):
```

Comment:
@MimiOlayeye Were you able to figure out how to delete these files? If you're having trouble merging dev and getting rid of the files something you can do is create a new branch stemming from dev and just copy and paste your code into new files so that there's no merge conflicts?